src/telemetry/telem_crsf.c

```diff
@@ -89,7 +89,7 @@ const char * _crsf_short_name(char *str, u8 telem)
     case TELEM_CRSF_TX_POWER: strcpy(str, _tr("Power")); break;
     case TELEM_CRSF_RX_QUALITY: strcpy(str, _tr("rxQual")); break;
     case TELEM_CRSF_TX_QUALITY: strcpy(str, _tr("txQual")); break;
-    case TELEM_CRSF_BATT_VOLTAGE: sprintf(str, _tr("Volt")); break;
+    case TELEM_CRSF_BATT_VOLTAGE: strcpy(str, _tr("Volt")); break;
```

Comment:
Lint Error:
```
92:  Almost always, snprintf is better than strcpy  
```

Reply:
google is wrong here.  snprintf is the worst option for us.  strlcpy would be better, but unneeded here.  we should disable this rule in the linter

Reply:
i would like to leave strcpy here for now. We should cleanup telemetry code later.